packages/opc-header/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
```suggestion
node_modules
dist
```

Reply:
@deshmukhmayur The dist file is to be committed as it has the final bundle

Reply:
@hybridx Wait What? Why?

Reply:
@sayak-sarkar fixed it. 😅